internal/pipe/aur/tmpl.go

```diff
@@ -0,0 +1,93 @@
+package aur
+
+type templateData struct {
+	Name            string
+	Desc            string
+	Homepage        string
+	Version         string
+	License         string
+	ReleasePackages []releasePackage
+	Maintainer      string
+	Contributors    []string
+	Provides        []string
+	Conflicts       []string
+	Depends         []string
+	OptDepends      []string
+	Arches          []string
+	Rel             string
+	Package         string
+}
+
+type releasePackage struct {
+	DownloadURL string
+	SHA256      string
+	Arch        string
+}
+
+const pkgBuildTemplate = `# This file was generated by GoReleaser. DO NOT EDIT.
+
+{{- with .Maintainer }}
+# Maintainer: {{ . }}
+{{- end }}
+{{- range .Contributors }}
+# Contributor: {{ . }}
+{{- end }}
+
+pkgname='{{ .Name }}'
+pkgver={{ .Version }}
+pkgrel={{ .Rel }}
+pkgdesc='{{ .Desc }}'
+url='{{ .Homepage }}'
+arch=({{ pkgArray .Arches }})
+license=('{{ .License }}')
+{{- with .Provides }}
+provides=({{ pkgArray . }})
+{{- end }}
+{{- with .Conflicts }}
+conflicts=({{ pkgArray . }})
+{{- end }}
+{{- with .Depends }}
+depends=({{ pkgArray . }})
+{{- end }}
+{{- with .OptDepends }}
+optdepends=({{ pkgArray . }})
+{{- end }}
+
+{{ range .ReleasePackages -}}
+source_{{ .Arch }}=('{{ .DownloadURL }}')
+sha256sums_{{ .Arch }}=('{{ .SHA256 }}')
+{{ printf "" }}
+{{ end }}
+
+{{-  with .Package -}}
+package() {
+{{ fixLines . }}
+}
+{{ end }}`
+
+const srcInfoTemplate = `pkgbase = {{ .Name }}
+	pkgdesc = {{ .Desc }}
+	pkgver = {{ .Version }}
+	pkgrel = {{ .Rel }}
+	url = {{ .Homepage }}
+	license = {{ .License }}
```

Comment:
The template doesn't generate a valid `.SRCINFO` file. All fields should be with a newline delimiter and beyond this point it seems like this template will gobble up any newlines past `license`? The test data is incorrect for reference.

Reply:
yup, fixed!